src/styles/styles.js

```diff
@@ -1782,6 +1782,7 @@ const styles = {
         paddingTop: 2,
         paddingBottom: 2,
         height: CONST.EMOJI_PICKER_ITEM_HEIGHT,
+        ...userSelect.userSelectNone,
```

Comment:
This is to avoid selecting text on long-press. I don't think we need to apply this only for mobile as long as we don't need to select emoji characters on EmojiPicker
Please let me know your thought